extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -583,6 +583,86 @@ oppia.directive('graphViz', function() {
 
 oppia.factory('graphUtilsService', [function() {
   return {
+    GRAPH_ADJACENCY_MODE: {
+      DIRECTED: 'directed',
+      INVERTED: 'inverted',
+      UNDIRECTED: 'undirected'
+    },
+    DFS_STATUS: {
+      VISITED: 'visited',
+      UNVISITED: 'unvisited',
+      STILL_VISITING: 'still visiting'
+    },
```

Comment:
I would leave spaces between all of the entries in this dictionary, in particular between the function declarations.